content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -1,9 +1,15 @@
 ---
-title: Node.js
+title: TypeScript/ Node.js
 aliases: [/docs/js/getting_started/nodejs]
 ---
 
-This guide will show you how to get started with tracing in Node.js.
+This guide will show you how to get started with tracing in TypeScript and Node.js.
```

Comment:
```suggestion
This guide will show you how to get started with tracing in Node.js.
```

WDYT @svrnm?

Reply:
yes, I think that makes sense, because at the end nodejs is used